integration_tests/test_math.py

```diff
@@ -106,6 +106,12 @@ def test_ceil():
     assert i == -13
 
 
+def test_remainder():
```

Comment:
We should run this test inside `check()`.

Reply:
Good catch, added it to check.